inventory/dynamic/azure/hosts.py

```diff
@@ -0,0 +1,862 @@
+#!/usr/bin/env python
```

Comment:
just checking for my understanding - is this file stock?

Reply:
What do you mean by stock? It's taken from https://github.com/ansible/ansible/blob/devel/contrib/inventory/azure_rm.py without any modification.

Reply:
I think the only thing I need to do here is add the ignore pylint/flake lines.

Reply:
possibly adding a comment that the file is otherwise identical to https://github.com/ansible/ansible/blob/devel/contrib/inventory/azure_rm.py would be handy